.travis.yml

```diff
@@ -90,6 +77,9 @@ jobs:
     - name: "ARCH=arm64 LD=ld.lld REPO=4.19"
       env: ARCH=arm64 LD=ld.lld REPO=4.19
       if: type = cron
+    - name: "ARCH=i386 LD=ld.lld REPO=4.19"
+      env: ARCH=i386 LD=ld.lld REPO=4.19
```

Comment:
FWIW, I think that we should focus on mainline and next first. Carrying out of tree patchsets for 4 different kernel versions is not very appealing. The chance of conflicts is probably low but churn still happens. If Android decides to carry these patches out of tree, great, we can just target our CI at those branches, which we are not responsible for maintaining.

Just my two cents though, not going to kick and scream about it :)

Reply:
No preference. If you (@tpimh ) keep the patches in another branch, I'll cherry pick them into Android with your name on them.

Reply:
Now only building linux an linux-next. If the CI hits green, I'll remove the last commit from the branch.